app/views/sufia/homepage/_sortable_featured.html.erb

```diff
@@ -1,6 +1,6 @@
 <% presenter = f.object.presenter %>
 <li class="dd-item dd3-item featured-item" data-id="<%= presenter.id %>">
-  <div class="dd-handle dd3-handle"><%= t :drag %></div>
+  <div class="dd-handle dd3-handle"><%= t 'sufia.sort_label' %></div>
```

Comment:
No i18n key was added here: https://github.com/projecthydra/sufia/commit/844e582f66e76ba4832f0314bc964dbff357d544 but as this was copypasta from Spotlight, I think we ought to add the key from spotlight: https://github.com/projectblacklight/spotlight/blob/master/config/locales/spotlight.en.yml#L3